content/references/protocol-reference/transactions/transaction-types/ammdeposit.md

```diff
@@ -0,0 +1,131 @@
+---
+html: ammdeposit.html
+parent: transaction-types.html
+blurb: Deposit funds into an Automated Market Maker in exchange for LPTokens.
+labels:
+  - AMM
+status: not_enabled
+---
+# AMMDeposit
+[[Source]](https://github.com/gregtatcam/rippled/blob/amm-core-functionality/src/ripple/app/tx/impl/AMMDeposit.cpp "Source")
+<!-- TODO: Update source link to merged version when available -->
+
+{% include '_snippets/amm-disclaimer.md' %}
+
+Deposit funds into an Automated Market-Maker (AMM) instance and receive the AMM's liquidity provider tokens (_LP Tokens_) in exchange. You can deposit one or both of the assets in the AMM's pool.
```

Comment:
```suggestion
Deposit funds into an Automated Market-Maker (AMM) instance and receive the AMM's liquidity provider tokens (_LP Tokens_) in exchange. You can deposit either asset in the AMM's pool.
```

Reply:
The intended meaning is you can deposit A, B, or A+B (and the semantics of A+B are different than depositing just one). Suggested wording sounds like the options are A or B (single-asset deposit) only.

Reply:
Misread the original. Keep as is.